### PR TITLE
feat(WEB-7418): Add 404 message

### PR DESCRIPTION
**Pull request by technicalcapt**
Follow up from this ticket:
- https://martide.atlassian.net/browse/WEB-7418

### CODE REVIEW

**Review by dgross881 — APPROVED** [review]

File: lib/literature/views/error_view.ex [hunk 1]
```diff
@@ -13,4 +14,68 @@ defmodule Literature.ErrorView do
...
+    """
+  end
+
+  def error_page(assigns) do
+    ~H"""
+    <div
+      class="error-content py-12 flex flex-col bg-white"
+      id="error-page-component"
+    >
+      <main class="flex-grow flex flex-col justify-center max-w-7xl w-full mx-auto px-4 sm:px-6 lg:px-8">
```
Comment (samhamilton) [inline lib/literature/views/error_view.ex:34]: @technicalcapt this should not be martide specific so can we please remove and just use the phoenix default?

Comment (technicalcapt) [reply lib/literature/views/error_view.ex:34]: @samhamilton Do you mean to remove the martide icon below in the error content?

Comment (technicalcapt) [reply lib/literature/views/error_view.ex:34]: https://github.com/martide/literature/pull/128/files#diff-3b814f6c184626a7bc0753d5cecf97be7cb5ad815c4d10da7fa05d0296b29eccR35-R40

Comment (samhamilton) [reply lib/literature/views/error_view.ex:34]: it should just be using the built in page and not report an error to sentry

Comment (technicalcapt) [reply lib/literature/views/error_view.ex:34]: Yes, when we're adding this error, it should show the 404 page and not report to Sentry.

**Pull request merged**